ph5/utilities/tests/test_ph5validate.py

```diff
@@ -61,30 +61,36 @@ def test_check_array_t(self):
         self.assertEqual(resp_check_info, self.resp_check_info)
 
     def test_check_response_t(self):
+        errors = {'array 004, station 0407, channel -2, response_table_n_i -1:'
+                  ' Metadata response with n_i=-1 has no response data.'}
         with LogCapture() as log:
-            log.setLevel(logging.ERROR)
+            log.setLevel(logging.WARNING)
             ret = self.ph5validate.check_response_t(self.resp_check_info)
-            self.assertEqual(ret[0].error, set())
+            self.assertEqual(set(ret[0].error), errors)
```

Comment:
This I added error message for n_i=-1. This is the case when metadatatoph5.py catches that the response has no signal.

Reply:
It seems correct for the `n_i=-1` case to be and Error exception rather than a warning. If there is no response information in a PH5 experiment than that experiment should be flagged, and should not output stationxml metadata until the experiment is fixed. It is not acceptable to be serving response-less PH5 data. The SEED archive works similar to this. PH5 expirments must be complete for them to be available to the public. This logic will also improve the PH5 validation process.

Reply:
In ph5tostationxml n_i=-1 will throw error if flag -E is not selected. However in ph5validate it should be logged as error because users want to see as most error as possible to fix it. If you want to change this to throw an error, we should discuss with DG.

Reply:
The exception handling should be different in ph5tostationxml and ph5validate. PH5toSationxml should be designed to output complete and correct metadata. It should not be designed as a testing tool. Ph5validate seems like it should be a data creation and testing tool so we would want it to run to completion and log errors rather than throwing program stopping exceptions.

Reply:
It is that way now. I have flag -E for debugging. It also help me to write unittest for warning when I use ph5/test_data/ph5/master.ph5 for testing. That file has the case n_i=-1. If there is no way to bypass that case, I cannot test the warning for response file names.
Flag -E will not affect user work because they will not use that in normal case.

Reply:
It seems fine to have the flag -E case as a backdoor for testing.